core/ui.libs/src/style/dapp-wrapper.scss

```diff
@@ -85,7 +85,7 @@ $sidebar-item-width: 40px;
     }
 
     .dapp-wrapper-sidebar-2-wrapper {
-      padding: 44px 0;
+      padding: 40px 0;
```

Comment:
Please also adjust the `evan-content-container` and `dapp-wrapper-sidebar` padding to keep everything correct aligned.